lib/javatests/com/google/android/material/datepicker/DateStringsTest.java

```diff
@@ -56,13 +56,13 @@ public void frYearMonthDayString() {
     startDate = setupLocalizedCalendar(locale, CURRENT_YEAR, 10, 30);
     assertThat(
         DateStrings.getYearMonthDay(startDate.getTimeInMillis(), locale),
-        is("30 nov. " + CURRENT_YEAR));
+        is("29 nov. " + CURRENT_YEAR));
```

Comment:
Not sure I understand the reason for changing these dates, is it a timezone issue?

Reply:
Hi, @paulfthomas do you have any feedback/suggestions regarding this change?

Reply:
Hi @kavitamp, we are looking into some other i18n issues that may be related. Thanks again for your contribution!